### PR TITLE
adding users to user function

### PR DESCRIPTION
**Pull request by raecasey**
added Yujing Huang and Thomas Mc Keon

### CODE REVIEW

File: ififuncs.py [hunk 1]
```diff
@@ -735,6 +735,12 @@ def get_user():
     elif user == '7':
         user = 'Eoin O\'Donohoe'
         time.sleep(1)
+    elif user == '8':
+        user = 'Yujing Haung'
```
Comment (kieranjol) [inline ififuncs.py]: typo, should be Huang ;)

Comment (raecasey) [reply ififuncs.py]: cool, thanks for catching that.
how do i change it? i know i should know, - sorry.

Comment (kieranjol) [reply ififuncs.py]: can't remember off the top of my head, you either update directly in your patch-6 here https://github.com/raecasey/IFIscripts/blob/patch-6/ififuncs.py#L739 or else you do it via this PR.
If stuck, I can come over and take a look? https://github.com/kieranjol/IFIscripts/pull/337/files

**Comment by raecasey** [comment]
hi - so i've corrected the patch-6 branch
can you check if that was the right thing to do and if so i guess it's ready to merge.
thanks

**Comment by raecasey** [comment]
have tested this - it works. - ready to merge
R

**Comment by kieranjol** [comment]
Awesomooooo thx!

**Pull request merged**